### ISSUE TITLE
Kubeflow installation using kfctl_k8s_istio fails at seldon-core-operator step

### ISSUE THREAD

**Issue by nsriram13** [kubeflow/kubeflow#4192]
/kind bug

**What steps did you take and what happened:**
I am following the instructions at the Kubeflow docs [here](https://www.kubeflow.org/docs/started/workstation/getting-started-linux/#microk8s) to install Kubeflow on my Linux server on micro-k8s. After ensuring that micro-k8s is working on my server, I followed through the docs link to [here](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/) to install Kubeflow using `kfctl_k8s_istio` config.

The installation runs fine till it tries to install the `seldon-core-operator` manifest at which point it fails. I have the error message I get below.

```
INFO[0008] creating Application/seldon-core-operator     filename="kustomize/kustomize.go:447"
Error: couldn't apply KfApp:  (kubeflow.error): Code 500 with message: kfApp Apply failed for kustomize:  (kubeflow.error): Code 500 with message: couldn't create resources from seldon-core-operator Error: Application.app.k8s.io "seldon-core-operator" is invalid: []: Invalid value: map[string]interface {}{"metadata":map[string]interface {}{"name":"seldon-core-operator", "namespace":"kubeflow", "creationTimestamp":"2019-09-26T20:46:49Z", "labels":map[string]interface {}{"app.kubernetes.io/version":"v0.6", "app.kubernetes.io/component":"seldon", "app.kubernetes.io/instance":"seldon-core-operator", "app.kubernetes.io/managed-by":"kfctl", "app.kubernetes.io/name":"seldon-core-operator", "app.kubernetes.io/part-of":"kubeflow"}, "generation":1, "uid":"c350eacc-e09e-11e9-bc5e-54bf646b0fc8"}, "spec":map[string]interface {}{"icons":interface {}(nil), "type":"seldon-core-operator", "version":"v1alpha2", "links":[]interface {}{map[string]interface {}{"description":"Docs", "url":"https://docs.seldon.io/projects/seldon-core/en/v0.3.0/"}}, "maintainers":[]interface {}{map[string]interface {}{"email":"dev@seldon.io", "name":"Seldon"}}, "owners":[]interface {}{map[string]interface {}{"email":"dev@seldon.io", "name":"Seldon"}}, "componentKinds":[]interface {}{map[string]interface {}{"group":"apps/v1", "kind":"StatefulSet"}, map[string]interface {}{"group":"v1", "kind":"Service"}, map[string]interface {}{"kind":"Deployment", "group":"extensions/v1beta1"}, map[string]interface {}{"kind":"Secret", "group":"v1"}, map[string]interface {}{"group":"v1", "kind":"ConfigMap"}}, "description":"Seldon allows users to create ML Inference Graphs to deploy their models and serve predictions", "keywords":[]interface {}{"seldon", "inference"}}, "apiVersion":"app.k8s.io/v1beta1", "kind":"Application"}: validation failure list:
spec.owners in body must be of type string: "object"
spec.maintainers in body must be of type string: "object"
Usage:
  kfctl apply [all(=default)|k8s|platform] [flags]

Flags:
  -h, --help      help for apply
  -V, --verbose   verbose output default is false

couldn't apply KfApp:  (kubeflow.error): Code 500 with message: kfApp Apply failed for kustomize:  (kubeflow.error): Code 500 with message: couldn't create resources from seldon-core-operator Error: Application.app.k8s.io "seldon-core-operator" is invalid: []: Invalid value: map[string]interface {}{"metadata":map[string]interface {}{"name":"seldon-core-operator", "namespace":"kubeflow", "creationTimestamp":"2019-09-26T20:46:49Z", "labels":map[string]interface {}{"app.kubernetes.io/version":"v0.6", "app.kubernetes.io/component":"seldon", "app.kubernetes.io/instance":"seldon-core-operator", "app.kubernetes.io/managed-by":"kfctl", "app.kubernetes.io/name":"seldon-core-operator", "app.kubernetes.io/part-of":"kubeflow"}, "generation":1, "uid":"c350eacc-e09e-11e9-bc5e-54bf646b0fc8"}, "spec":map[string]interface {}{"icons":interface {}(nil), "type":"seldon-core-operator", "version":"v1alpha2", "links":[]interface {}{map[string]interface {}{"description":"Docs", "url":"https://docs.seldon.io/projects/seldon-core/en/v0.3.0/"}}, "maintainers":[]interface {}{map[string]interface {}{"email":"dev@seldon.io", "name":"Seldon"}}, "owners":[]interface {}{map[string]interface {}{"email":"dev@seldon.io", "name":"Seldon"}}, "componentKinds":[]interface {}{map[string]interface {}{"group":"apps/v1", "kind":"StatefulSet"}, map[string]interface {}{"group":"v1", "kind":"Service"}, map[string]interface {}{"kind":"Deployment", "group":"extensions/v1beta1"}, map[string]interface {}{"kind":"Secret", "group":"v1"}, map[string]interface {}{"group":"v1", "kind":"ConfigMap"}}, "description":"Seldon allows users to create ML Inference Graphs to deploy their models and serve predictions", "keywords":[]interface {}{"seldon", "inference"}}, "apiVersion":"app.k8s.io/v1beta1", "kind":"Application"}: validation failure list:
spec.owners in body must be of type string: "object"
spec.maintainers in body must be of type string: "object"
```

**What did you expect to happen:**
The expectation was for the installation to run end to end as described in the docs.

**Anything else you would like to add:**
My suspicion is that the `owners` and `maintainers` is set to NULL or non-string in the manifest and this causes failure. Not sure how to verify/fix this though.
```
spec.owners in body must be of type string: "object"
spec.maintainers in body must be of type string: "object"
```

**Environment:**

- Kubeflow version: `v0.6.2`
- kfctl version: `v0.6.2-0-g47a0e4c7`
- Kubernetes platform: `microk8s`
- Kubernetes version: 
```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.9", GitCommit:"e09f5c40b55c91f681a46ee17f9bc447eeacee57", GitTreeState:"clean", BuildDate:"2019-05-27T16:08:57Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.9", GitCommit:"e09f5c40b55c91f681a46ee17f9bc447eeacee57", GitTreeState:"clean", BuildDate:"2019-05-27T15:58:45Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
- OS: `Ubuntu 18.04.3 LTS`

**Comment by niya3** [comment]
@nsriram13 here is a hotfix

```diff
diff --git a/kustomize/seldon-core-operator/overlays/application/application.yaml b/kustomize/seldon-core-operator/overlays/application/application.yaml
index 3812cfb..7f75ff5 100644
--- a/kustomize/seldon-core-operator/overlays/application//application.yaml
+++ b/kustomize/seldon-core-operator/overlays/application//application.yaml
@@ -19,11 +19,9 @@ spec:
   description: "Seldon allows users to create ML Inference Graphs to deploy their models and serve predictions"
   icons:
   maintainers:
-    - name: Seldon
-      email: dev@seldon.io
+   - "Seldon (dev@seldon.io)"
   owners:
-    - name: Seldon
-      email: dev@seldon.io
+   - "Seldon (dev@seldon.io)"
   keywords:
    - "seldon"
    - "inference"
```

**Comment by jlewi** [comment]
We just released 0.7.0 which fixes a number of issues.
Please try the latest release and if your still having problems please open a new issue.

Note that Kubeflow is currently tested on Kubernetes 1.14 and known not to work on 1.16.
